tests/integ/modin/frame/test_iloc.py

```diff
@@ -1791,8 +1809,8 @@ def test_df_iloc_set_with_row_key_list(
             item_values,
             item_values,
             wrap_item="na",
-            wrap_row="na",
-            wrap_col="na",
+            wrap_row="index",
```

Comment:
can you keep using na then adjust keys in `wrap_key_as_expected_type` later?

Reply:
There is other functionality for na? na handles the else case so I didnt want to change it